ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2

```diff
@@ -8,6 +8,49 @@ router isis {{ router_isis.instance }}
 {%     if router_isis.is_type is arista.avd.defined %}
    is-type {{ router_isis.is_type }}
 {%     endif %}
+{%     for redistribute_route in router_isis.redistribute_routes | arista.avd.natural_sort('source_protocol') %}
+{%         if redistribute_route.source_protocol is arista.avd.defined %}
+{%             set redistribute_route_cli = "redistribute " ~ redistribute_route.source_protocol %}
+<<<<<<< Updated upstream
```

Comment:
Need to resolve these lines. Should not present in templates.

Reply:
Resolved the accidental merge insert along with aforementioned suggestions in next commit